elfeed-org.el

```diff
@@ -36,6 +36,8 @@
 (require 'org)
 (require 'dash)
 (require 's)
+(require 'cl-lib)
+(require 'xml)
```

Comment:
Have you considered to add these packages to the `Package-Requires` header on line 6?  Or are these always available?